src/domain/user/user_entity.go

```diff
@@ -0,0 +1,58 @@
+package user
+
+import (
+	"fmt"
+
+	"github.com/yuuki-tsujimura/architecture-study/src/domain/id"
+	"github.com/yuuki-tsujimura/architecture-study/src/usecase"
+)
+
+type User struct {
+	UserId   id.UserId
+	Name     string
+	Password Password
+	Profile  string
+	Career   Career
+	Skills   []Skill
```

Comment:
```suggestion
	Skills   []*Skill
```
構造体は基本的にはポインタで持っておくと良いです。

構造体のスライスで持つと、それだけメモリを食うことになるため。

Reply:
ポインタに変更しました。